content/es/01_friendly-introduction-to-zero-knowledge.md

```diff
@@ -0,0 +1,550 @@
+---
+title: 'Una introducción amigable a Zero Knowledge'
+date: '2023-07-17'
+tags: ['zero-knowledge']
+draft: false
+layout: PostSimple
+slug: "friendly-introduction-to-zero-knowledge"
+images: ['../assets/01_zkp-magic.png']
+summary: "Las pruebas de conocimiento cero（Zero Knowledge Proofs）son mágicas. Nos permiten hacer cosas que antes ni siquiera podíamos imaginar. Este es el primero de una serie de artículos sobre pruebas de conocimiento cero y sus aplicaciones. Veremos qué son, por qué importan, cómo funcionan y dónde pueden usarse."
+---
+
+_Este artículo fue traducido por Gelois, Alex y [Yago Pajariño](https://x.com/0xyago)_
```

Comment:
```suggestion
_Este artículo fue traducido por Gelois, Alex y Yago Pajariño_
```

I have tried to keep external links to a minimum in the main body (only external links appears in Reference section right now), hope that's OK! I suggest:
- Keep external link in Acknowledgements section - mind adding it there? Similar to the bt-pr one https://github.com/zkintro/zkintro/pull/8/files#diff-781053a6d0750851658a297e9259761e5937f6aefbf06e71a35ce530b17ce235R471
- In tweet announcement of translation mention translators